apps/mobile/src/navigation/MainTabNavigator/TabBar.tsx

```diff
@@ -45,17 +47,21 @@ function TabItem({ navigation, route, icon, activeRoute }: TabItemProps) {
       accessibilityRole="button"
       accessibilityState={isFocused ? { selected: true } : {}}
       activeOpacity={1}
-      className={clsx(
-        `px-0 flex h-8 w-8 items-center justify-center rounded-full active:bg-faint dark:active:bg-[#2B2B2B]`,
-        {
-          'border border-black dark:border-white': isFocused,
-        }
-      )}
+      className="pt-3 px-4 flex items-center justify-center"
       eventElementId="Navigation Tab Item"
       eventName="Navigation Tab Item Clicked"
       properties={{ variant: 'Main', route: route.name }}
+      onPressIn={() => setIsPressed(true)}
+      onPressOut={() => setIsPressed(false)}
```

Comment:
@tbezman Adding this hack to maintain the same `pressed` ui behaviour from `active:bg-faint dark:active:bg-[#2B2B2B]`

Tried with tailwindcss [group](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state) and it seems not stable enough in nativewind. 

Reference issue:
https://github.com/marklawlor/nativewind/issues/233